department-of-veterans-affairs/caseflow
# Paula/Caseflow-535 Timeliness failure for edit NOD date

Resolves [CASEFLOW_535](https://vajira.max.gov/browse/CASEFLOW-535)

### Description
Add timeliness error message for edit NOD date modal

### Acceptance Criteria
- [ ] Failure message appears if changing NOD date affects timeliness of any issues 

### Testing Plan
1. Go to veteran file 152003980
2. Click 'Edit NOD link' and change nod date to one week after original
3. Select a change reason
4. Click submit
5. Verify that unaffected and unaffected issues appear in modal and replace the form in the modal

### User Facing Changes

 BEFORE|AFTER
 ---|---
<> | <img width="979" alt="Screen Shot 2021-02-17 at 7 18 26 PM" src="https://user-images.githubusercontent.com/68879427/108300323-f8c84c80-7154-11eb-92ff-4addaf359a49.png">
